UI/Source/CollectionViews/mac/CollectionViewController.swift

```diff
@@ -426,7 +426,7 @@ open class CollectionViewController: NSViewController, CollectionViewDelegate {
             case .notLoaded:
                 break
             case .loading(let models):
-                if models == nil {
+                if models == nil || state.isEmpty {
```

Comment:
@wkiefer I'm not sure this is always the desired behaviour. Having the spinner only shown when `nil` allowed the ModelCollection some degree of control over what "loading" means. Either...

1) I'm clearing the current state and want to reload the collection view showing the spinner.
2) I've loaded an empty collection and want to silently populate the collection with new content without showing the spinner.

It seems a alternative to the above fix could be for ModelCollections wanting to show a spinner when empty to always provide .loading(nil) when empty...